### PR TITLE
Enhance issue #5507 Add switch button and pages layout for fetchers/websearch

### PR DESCRIPTION
**Pull request by JunyuLuo123**
Enhances #5507

Add the switch button and pages layout.

### CODE REVIEW

**Comment by JunyuLuo123** [comment]
image
While the page is in 1. The previous button will not be access.

File: src/main/java/org/jabref/logic/importer/fileformat/MedlineImporter.java [hunk 1]
```diff
@@ -55,6 +55,17 @@ public class MedlineImporter extends Importer implements Parser {
...
+
+    public MedlineImporter() {
+        this.xmlInputFactory = XMLInputFactory.newInstance();
+        // prevent xxe (https://rules.sonarsource.com/java/RSPEC-2755)
+        xmlInputFactory.setProperty(XMLConstants.ACCESS_EXTERNAL_SCHEMA, "");
+        // required for reading Unicode characters such as &#xf6;
+        xmlInputFactory.setProperty(XMLInputFactory.IS_COALESCING, true);
+        // TODO: decide if necessary, if disabled MedlineImporterTestNbib fails
+        xmlInputFactory.setProperty(XMLInputFactory.IS_SUPPORTING_EXTERNAL_ENTITIES, true);
+    }
```
Comment (koppor) [inline src/main/java/org/jabref/logic/importer/fileformat/MedlineImporter.java:68]: Why this change?

File: src/main/resources/build.properties [hunk 2]
```diff
@@ -6,3 +6,6 @@ springerNatureAPIKey=${springerNatureAPIKey}
 astrophysicsDataSystemAPIKey=${astrophysicsDataSystemAPIKey}
 ieeeAPIKey=${ieeeAPIKey}
 biodiversityHeritageApiKey=${biodiversityHeritageApiKey}
+Previous=Create property
+Next=Create property
+Page\:=Create property
```
Comment (koppor) [inline src/main/resources/build.properties:11]: I don't understand this? -- Did you modify the wrong file? `JabRef_en.properties`. See https://devdocs.jabref.org/code-howtos/localization.html for more information

File: src/main/java/org/jabref/gui/importer/ImportEntriesDialog.java [hunk 3]
```diff
@@ -49,13 +50,16 @@
 import jakarta.inject.Inject;
 import org.controlsfx.control.CheckListView;
 
+@SuppressWarnings("checkstyle:RegexpMultiline")
```
Comment (koppor) [inline src/main/java/org/jabref/gui/importer/ImportEntriesDialog.java:53]: Why this?

**Review by koppor — CHANGES_REQUESTED** [review]
The PR is far from complete. The connection to the fetchers is not there. This is more complicated than just adding a button somewhere in the dialog

File: src/main/java/org/jabref/logic/importer/fileformat/MedlineImporter.java (hunk 1 above)
Comment (JunyuLuo123) [reply src/main/java/org/jabref/logic/importer/fileformat/MedlineImporter.java:68]: I got no idea, I didn't modify this part.

File: src/main/resources/build.properties (hunk 2 above)
Comment (JunyuLuo123) [reply src/main/resources/build.properties:11]: When I create a button name and page layout it will show error if I didn't do that.

File: src/main/java/org/jabref/gui/importer/ImportEntriesDialog.java (hunk 3 above)
Comment (JunyuLuo123) [reply src/main/java/org/jabref/gui/importer/ImportEntriesDialog.java:53]: It changing automaticity. I didn't do anything on that.

**Comment by JunyuLuo123** [comment]
Sorry about that. Cause the time limit that's what I can think and done.

**Comment by calixtus** [comment]
Hi @JunyuLuo123 , what's the status here? Are you planning to continue this PR?

**Comment by calixtus** [comment]
As the user obviously lost interest in this PR, i'm closing it for housekeeping. Feel free to open a new PR if you decide to continue your work.
